ci/libbuild.sh

```diff
@@ -43,10 +43,21 @@ install_builddeps() {
         dnf builddep -y $pkg
     else
         yum install -y make rpm-build
-        yum-builddep -y rpm-ostree
+        # XXX: to remove right after
+        # https://github.com/CentOS/sig-atomic-buildscripts/pull/301
+        curl -LO https://src.fedoraproject.org/cgit/rpms/rpm-ostree.git/plain/rpm-ostree.spec
+        yum-builddep -y ./rpm-ostree.spec
     fi
 
     # builddeps+runtime deps
-    yum install -y $pkg
-    rpm -e $pkg
+    # XXX: to remove right after
+    # https://github.com/CentOS/sig-atomic-buildscripts/pull/301
+    local id=$(. /etc/os-release && echo $ID)
+    if [ "$id" == centos ]; then
+      yum install -y $pkg-client
```

Comment:
I don't think that's going to work...`rpm-ostree-client` isn't in the repos we enable, right?

Reply:
Bleh, you're right. Should've checked. I picked up https://github.com/CentOS/sig-atomic-buildscripts/blob/master/rhel-atomic-rebuild.repo, which seems like it has a decently recent rpm-ostree. Tested working in a container!